src/Core/Termination/Positivity.idr

```diff
@@ -166,6 +166,7 @@ calcPositive loc n
          logC "totality.positivity" 6 $ do pure $ "Calculating positivity: " ++ show !(toFullNames n)
          case !(lookupDefTyExact n (gamma defs)) of
               Just (TCon _ _ _ _ _ tns dcons _, ty) =>
+                  let dcons = fromMaybe [] dcons in
```

Comment:
Does it make sense to add a test case ensuring that this cannot be exploited
(e.g. by trying to have positivity be computed in one module & reused in another
after adding constructors):

```idris2
module OopsDef

%default total

public export
data Oops : Type

forcePosCheck : Oops -> Oops
forcePosCheck x = x
```

and

```idris
import OopsDef

%default total

failing "Oops is not total, not strictly positive"

  data OopsDef.Oops : Type where
    MkFix : Not Oops -> Oops
```